.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    permissions: read-all
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [16, 14, 12, 10]
+        os: [windows-2019, macos-latest, ubuntu-latest]
+        arch: [x64]
+        include:
+          - os: windows-2019
+            arch: x86
+            node: 16
+          - os: windows-2019
+            arch: x86
+            node: 14
+          - os: windows-2019
+            arch: x86
+            node: 12
+          - os: windows-2019
+            arch: x86
+            node: 10
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+        # Needed to fix node-webkit prebuilds, see https://github.com/nwjs/node/issues/46
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        with:
+          python-version: 2.7
```

Comment:
prebuild currently doesn't work with Python 3.11 because it uses an old version of node-gyp. Python 3.10 should work.

https://github.com/prebuild/prebuild/issues/286